projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss

```diff
@@ -2,27 +2,27 @@
 
 :host {
 
-  
+
   --wvr-dropdown-menu-animation-speed: 0.25s;
   --wvr-dropdown-menu-background: var(--wvr-white);
   --wvr-dropdown-menu-border: 1px solid var(--wvr-gray-dark);
   --wvr-dropdown-menu-border-radius: 6px;
   --wvr-dropdown-menu-display: flex;
   --wvr-dropdown-menu-flex-direction: column;
   --wvr-dropdown-menu-padding: 25px;
-  --wvr-dropdown-menu-width: 200px;
+  --wvr-dropdown-menu-width: fit-content;
   --wvr-dropdown-menu-x-offset: 0px;
   --wvr-dropdown-menu-y-offset: -1px;
   --wvr-dropdown-menu-item-margin: 0 0 10px 0;
-  
+
 }
 
 .wvr-dropdown {
 
   cursor: pointer;
   font-family: var(--wvr-font-family-sans-serif);
   position: relative;
-  width: min-content;
```

Comment:
This should be `var(--wvr-dropdown-menu-width)`

Reply:
width value changed to var(--wvr-dropdown-menu-width)